src/integrationTest/java/uk/gov/hmcts/darts/testutils/stubs/wiremock/DartsGatewayStub.java

```diff
@@ -28,10 +29,11 @@ public void verifyDoesntReceiveDarEvent() {
 
     public void verifyReceivedNotificationType(int type) {
         var notificationType = "\"notification_type\":\"" + type + "\"";
-        wait(1000);
-        verify(exactly(1), postRequestedFor(urlEqualTo(DAR_NOTIFY_PATH))
-            .withRequestBody(containing(notificationType)));
-
+        waitForMax10SecondsWithOneSecondPoll(() -> {
```

Comment:
Do we need this in the method above also? - verifyDoesntReceiveDarEvent

Reply:
Kind of impossible i guess. How do we know when something is never going to happen. Only way is to wait the longest time and then check